src/v3/src/components/Images/DeviceIcon.tsx

```diff
@@ -10,40 +10,19 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+import { Theme } from '@mui/material';
 import { FunctionComponent, h } from 'preact';
 
 import { loc } from '../../util';
+import { DevicesIcon } from '../Icon/OdysseyIcons';
 
 export const DeviceIcon: FunctionComponent = () => (
```

Comment:
nit: Since we're defaulting to Odyssey whenever possible, can we have the name match the naming convention that Odyssey uses, so in this case renaming this component and file to also be `DevicesIcon` rather than `DeviceIcon`